lms/resources/_js_config.py

```diff
@@ -68,7 +68,7 @@ def add_document_url(self, document_url):
                 "path": self._request.route_path(
                     "blackboard_api.files.via_url",
                     course_id=self._request.params["context_id"],
-                    file_id=document_url.split("/")[3],
+                    _query={"file_id": document_url},
```

Comment:
`JSConfig` just passes the file ID directly from the DB (so in the `blackboard://...` form) to the frontend as a query param on the Via URL API call. The frontend just treats this ID as an opaque string, so while `blackboard://...` do flow through the frontend the frontend code doesn't know anything about them.

Reply:
We might want to name the query param (or path component) `document_url` rather than `file_id` since it's a `blackboard://...` URL